### ISSUE TITLE
Set up continuous integration (build server)

### ISSUE THREAD

**Issue by FRosner** [FRosner/DataGenerator#168]
I'd like to have a build server (hudson, jenkins) to build our software regularly from git. We have to make sure that the Swing GUI tests run as well.

**Comment by FRosner** [comment]
We should take a look at http://blog.dahanne.net/2011/07/18/run-ui-tests-on-a-headless-jenkins-hudson-continuous-integration-server-running-ubuntu/

**Comment by FRosner** [comment]
@Husterknupp do not supply credentials or anything like this here. This is visible to anyone. Not even the link to the build server, please. Transmit it to me personally (IM or mail is fine).

**Comment by Husterknupp** [comment]
I can see your point. I won't do anything like that.

**Comment by FRosner** [comment]
@Husterknupp how is it going? Took you some time. If there is trouble with the server you got right now, @3ll0b0 offered us to use his machine.

**Comment by Husterknupp** [comment]
Yeah, I got some first jobs at the jenkins machine up and running. But I am not ready, yet. I did not create any job with gui testing. When #178 is done there will be progress again.

**Comment by FRosner** [comment]
@Husterknupp if you want we can create a profile that does not execute GUI tests at all. That way you can at least set up the server to build DataGenerator. Just create me a ticket if you want me to create this additional profile. It is no big deal since the categories are introduced already.

**Comment by FRosner** [comment]
@Husterknupp I managed to build DataGenerator on Jenkins without GUI tests so far.

**Comment by FRosner** [comment]
Next steps:
- VNC server for GUI tests
- configure SMTP for e-mail notifications
